### ISSUE TITLE
Default behavior of Rule Multiline*BraceLayout since 0.40.0

### ISSUE THREAD

**Issue by obfuscoder** [bbatsov/rubocop#3210]
Starting with version 0.40.0 I get a rubocop checkstyle warning for code like this:

``` ruby
some_call(
  some_really_long_parameter)
```

The following warning appears: _Closing method call brace must be on the line after the last argument when opening brace is on a separate line from the first argument._

To fix the code I would need to put the closing brace on the newline:

``` ruby
some_call(
  some_really_long_parameter
)
```

I cannot put the first parameter on the same line as the opening brace as this would violate the maxmimum length of the line.

What is the reason to have this rule? I suppose the rule makes sense if the method has several arguments, but I don't see the point in putting the closing brace on a separate line when jsut having one parameter which does not fit on the first line.

**Comment by Drenmi** [comment]
This style is still available. Just configure your cop to use the `same_line` option. 🙂 

```yml
Layout/MultilineMethodCallBraceLayout:
  EnforcedStyle: same_line
```
